ci-operator/jobs/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14-periodics.yaml

```diff
@@ -731,3 +731,95 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 0 6 11 2 *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: release-v1.14
+    org: openshift-pipelines
+    repo: release-tests
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: openshift-pipelines-ocp4.17-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.17"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-pipelines-release-tests-release-v1.14-openshift-pipelines-ocp4.17-lp-interop-openshift-pipelines-interop-aws
```

Comment:
Here it should be `v1.15`

Reply:
@VeereshAradhya - Creating separate [PR#54134](https://github.com/openshift/release/pull/54134/files) for these changes to OpenShift Pipelines v1.15 on OCP 4.17.